sdks/python/apache_beam/examples/streaming_wordcount.py

```diff
@@ -46,8 +48,10 @@ def run(argv=None):
       '--output_topic', required=True,
       help='Output PubSub topic of the form "/topics/<PROJECT>/<TOPIC>".')
```

Comment:
Is this valid `/topics/<PROJECT>/<TOPIC>`. It looks like there is a difference in two runners in how this topic is used.